packages/cubejs-schema-compiler/src/scaffolding/ScaffoldingSchema.ts

```diff
@@ -36,21 +77,49 @@ const DRILL_MEMBERS_DICTIONARY = [
 const idRegex = '_id$|id$';
 
 export class ScaffoldingSchema {
-  constructor(dbSchema) {
+  private tableNamesToTables: any;
+
+  public constructor(private readonly dbSchema: Record<string, Record<string, any>>) {
     this.dbSchema = dbSchema;
```

Comment:
You declared property with `private readonly`, typescript will generate setter for it automatically. We can drop line

```
 this.dbSchema = dbSchema;
```